README.md

```diff
@@ -101,21 +101,14 @@ key is in this repo under [deb.asc](/deb.asc).
 To install eza from this repo use:
 
 ```bash
-wget -qO- https://raw.githubusercontent.com/eza-community/eza/main/deb.asc | sudo tee /etc/apt/trusted.gpg.d/gierens.asc
-echo "deb http://deb.gierens.de stable main" | sudo tee /etc/apt/sources.list.d/gierens.list
+sudo mkdir -p /etc/apt/keyrings
+wget -qO- https://raw.githubusercontent.com/eza-community/eza/main/deb.asc | sudo gpg --dearmor -o /etc/apt/keyrings/gierens.gpg
+echo "deb [signed-by=/etc/apt/keyrings/gierens.gpg] http://deb.gierens.de stable main" | sudo tee /etc/apt/sources.list.d/gierens.list
+sudo chmod o+r /etc/apt/keyring/gierens.gpg /etc/apt/sources.list.d/gierens.list
```

Comment:
Interesting, that's a good point ... I just wonder if a `sudo chmod 644 ...` might not be even better, just to make sure all permissions are set properly. What do you think?

Reply:
Sorry if I'm a little nitpicky on this, but since this is for some users a very crucial part of the documentation I would be great if we get every detail right this time :)

Reply:
done!
someone pointed out I was missing an `s` on `keyrings` so fixed that too

Reply:
Ah good catch! Thanks!